### PR TITLE
apple2_cass.xml: Add 四人麻雀

### PR DESCRIPTION
**Pull request by FakeShemp**
Dumped by Hubz.

### CODE REVIEW

File: hash/apple2_cass.xml [hunk 1]
```diff
@@ -1484,6 +1484,19 @@
 		</part>
 	</software>
 
+	<software name="mahjong">
+		<description>Shi-ri Mahjong</description>
```
Comment (cuavas) [inline hash/apple2_cass.xml]: Is this an official title used in the software or its documentation?  I'd prefer using standard romanisation if it isn't.

Comment (FakeShemp) [reply hash/apple2_cass.xml]: Cassette
No, and my Japanese is terrible. If you know more correct romanisation, then please don't hesitate to tell me or change it yourself.

**Comment by etabeta78** [comment]
I think it's 4-nin Mahjong, isn't it?

**Comment by FakeShemp** [comment]
Ah, "Yonin". Of course.

**Pull request merged**